SCClassLibrary/JITLib/ProxySpace/extStoreOn.sc

```diff
@@ -188,8 +188,15 @@
 					if (srcStr.notEmpty) { str = str ++ ".source_(" ++ srcStr ++ ")" };
 				} {
 					if (isInCurrent) { 	// ~out
-						if (srcStr.notEmpty) { str = str + "=" + srcStr };
-
+						if (srcStr.notEmpty) {
+							if (this.isKindOf(Ndef)) {
```

Comment:
we are 5 if's deep here -- this code is really difficult to understand, and should be refactored at some point

Reply:
Yes. For now this looks OK.

It is reasonable to keep all these string related cases in a single function. But maybe this test  in particular would be better dispatched via a method of the respective class, such like `asCodeWithObject`.

Reply:
exploding it out to methods on separate classes would be even more obfuscatory..

Reply:
not if we find good names for the methods, yes if we don't.

Reply:
going to resolve this and stop replying as it's too far OT.